pkg/utils/trace.go

```diff
@@ -0,0 +1,39 @@
+//
+// Copyright (c) 2018 The heketi Authors
+//
+// This file is licensed to you under your choice of the GNU Lesser
+// General Public License, version 3 or any later version (LGPLv3 or
+// later), or the GNU General Public License, version 2 (GPLv2), in all
+// cases as published by the Free Software Foundation.
+//
+
+package utils
+
+import (
+	"path/filepath"
+	"runtime"
+)
+
+// TraceSkip is like Trace, but additionally skips stack frames. So TaceSkip(0)
+// behaves like Trace(), TraceSkip(1) refers to the call of the current
+// function, TraceSkip(2) refers to the call of the calling function, and so on.
+func TraceSkip(skip int) (string, string, int) {
+	pc := make([]uintptr, 15)
```

Comment:
Is there a check required here? skip is less than or equal to 15?

Reply:
I would like an explanation for this question before the patch is merged.

Reply:
@raghavendra-talur  if you want to block this PR until this is solved, you should raise a review change request. :-)

Reply:
@obnoxXx @ansiwen thanks!